ppafm/ocl/cl/FF.cl

```diff
@@ -719,6 +848,31 @@ __kernel void interp_at(
 
 }
 
+// Raise all array elements to the same power. Negative values are set to zero.
```

Comment:
As I understand you do the formula `E = A  ( rho1 * rho2  )^b` step-by-step by calling new kernell for each operation. That is probably great for debugging, but I guess for performance it would be better to do it at once just at the end of the convolution (` rho1 * rho2` ) ? (but perhaps it is just very minor overhead)

Reply:
Aha sorry, I did not think about it deeply enouh. I is necessary to do the power `rho^b` first, and then convolution. Than it cannot be put int one kernel perhaps (?).

Reply:
Yes, I think you have to do it this way around. Although, I'm not sure what the Fourier transform does to it. Maybe there is some more clever way to do it.